src/azure-cli/install.sh

```diff
@@ -179,4 +180,15 @@ if [ "${use_pip}" = "true" ]; then
     fi
 fi
 
+if [[ "${AZ_DEV_VERSION}" -ne "none" ]]; then
+    echo "(*) Installing Azure Dev CLI"
+
+    curl -fsSLO https://aka.ms/install-azd.sh
+    chmod +x ./install-azd.sh
+
+    ./install-azd.sh --version $AZ_DEV_VERSION
```

Comment:
Can you add a test in https://github.com/devcontainers/features/blob/main/test/azure-cli/test.sh to verify that the `azd` CLI is installed properly? If you want to add multiple test cases to verify with and without `azd` installed, you can set up multiple scenarios with different sets of feature options following the example in https://github.com/devcontainers/features/tree/main/test/dotnet.

Reply:
Added a single case to the main test for `azure-cli`.

I could break it to multiple scenarios if you prefer (although I'm not sure how to run the tests)

Reply:
Thanks for adding the test case! Looks like the test is failing and it's not installed `azd` as part of the feature execution.

You should be able to reproduce locally by running the test with the `devcontainer features test` command like the CI does: https://github.com/devcontainers/features/blob/main/.github/workflows/test-all.yaml#L53-L57

Reply:
I'm not a bash expert, but I saw somewhere that you might not be able to use `-ne` with string comparisons. Try changing `-ne` to `!=` and see if the installation works / tests pass.

Reply:
Ah, should have looked for a workflow file to see how the tests work run.

I'll work on getting the devcontainer cli installed (was fighting that the other day due to the cascading dependencies, maybe I should try a devcontainer to run it in 😜)